src/Model/Linsen.hs

```diff
@@ -122,6 +122,7 @@ parse day =
                           []    -> fail "Failed to index into richtext"
                           (v:_) -> pure v)
                   >=> (.: "text")
+                  >=> filterM (pure . not . isSpace)
                   >=> \s ->
                     let sameDay = pure day == parseTime swedishTimeLocale "%A %d-%m-%Y" s ||
```

Comment:
Doesn't filtering out whitespace break the case where we parse weekdays?

Reply:
Myes

Reply:
Should be fixed now